torch/csrc/jit/mobile/register_mobile_ops.cpp

```diff
@@ -24,15 +24,132 @@ static auto registry0 = torch::RegisterOperators().op(
     return at::add(a, b, c);
   })
 ).op(
-  "_aten::_convolution",
-  torch::RegisterOperators::options().kernel(c10::TensorTypeId::CPUTensorId,
-  [](at::Tensor input, at::Tensor weight, c10::optional<at::Tensor> bias,
-  std::vector<int64_t> stride, std::vector<int64_t> padding,
-  std::vector<int64_t> dilation, bool transposed, std::vector<int64_t> output_padding,
-  int64_t groups, bool benchmark, bool deterministic, bool cudnn_enabled) {
-  return at::_convolution(input, weight, optional_to_tensor(bias), stride, padding, dilation,
-                     transposed, output_padding, groups, benchmark, deterministic, cudnn_enabled);
-  })
+"_aten::add_.Tensor",
+torch::RegisterOperators::options().kernel(c10::TensorTypeId::CPUTensorId,
+                                           [](at::Tensor a, at::Tensor b, at::Scalar c) ->at::Tensor {
```

Comment:
nit: space after '->' (I did a search in the codebase and it seems to be common style to have space on both sides)